### PR TITLE
Add Lyft recap post

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _posts/2022-11-30-trino-summit-2022-lyft-recap.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Trino for Large Scale ETL at Lyft"
```
Comment (mosabua) [inline _posts/2022-11-30-trino-summit-2022-lyft-recap.md:3]: Title case and date change please

File: _posts/2022-12-12-trino-summit-2022-lyft-recap.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+the volume of their workloads due to recent improvements in Trino and upgrades
+in Java versions. Lyft does not using fault-tolerant execution, but has started
+seeing interest in using Trino for ETL jobs due to the faster turnaround. Some
+issues Lyft has faced has been around how resource hungry Trino is, as well as,
+the issue where the coordinator can be a single point of failure for queries
+executing on a cluster.
+
+Lyft was one of the earliest companies to really push using Trino for ETL use
+cases. They built custom best effort rollback code in Airflow. If a query
+failed, the operation would revert to the state before the operation began. Lyft
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-lyft-recap.md:41]:
```suggestion
fails, the operation reverts to the state before the operation began. Lyft
```

File: _posts/2022-12-12-trino-summit-2022-lyft-recap.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+runs four ETL Trino clusters split by the type of workload used on that cluster.
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-lyft-recap.md:42]: so they are all ETL .. but still different workload?

File: _posts/2022-12-12-trino-summit-2022-lyft-recap.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+The best practices Lyft has found is careful usage around broadcast joins, query
```
Comment (mosabua) [inline _posts/2022-12-12-trino-summit-2022-lyft-recap.md:43]: practices are

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2022-12-12-trino-summit-2022-lyft-recap.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+author: "Charles Song, Ritesh Varyani, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/lyft.jpg
+---
+Buckle up, for the next [post in the Trino Summit 2022 recap series]({% post_url
+2022-11-21-trino-summit-2022-recap %}). In this post, we're covering the talk
+given by Lyft engineers, Charles and Ritesh, on how they have not only scaled
+Trino for more usage, but also with less nodes and more effective usage. They
+also started moving to utilizing Trino more for ETL rather than just interactive
+analytics. You don't want to miss your driver for this post.
```
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-12-12-trino-summit-2022-lyft-recap.md:13]: was trying to make a play on words for Lyft driver

[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-12-12-trino-summit-2022-lyft-recap.md:13]: fixed

File: _posts/2022-12-12-trino-summit-2022-lyft-recap.md (hunk 3 above)
Comment (bitsondatadev) [reply _posts/2022-12-12-trino-summit-2022-lyft-recap.md:42]: Nope, I removed ETL ahead of Trino

**Comment by bitsondatadev** [comment]
not sure why this is failing..it builds locally for me

**Comment by mosabua** [comment]
I get this error message locally @bitsondatadev 

```
 Generating...
          Skipping: _posts/2022-12-12-trino-summit-2022-lyft-recap.md has a future date
    Liquid Warning: Liquid syntax error (line 294): Unexpected character = in "{{ task_instance.xcom_pull(task_ids='task_1',key='return_value')[0] }}" in /Users/manfred/dev/github/simpligility/trino.io/_posts/2022-07-13-how-to-use-airflow-to-schedule-trino-jobs.md
  Liquid Exception: Could not find post "2022-12-12-trino-summit-2022-lyft-recap" in tag 'post_url'. Make sure the post exists and the name is correct. in /Users/manfred/dev/github/simpligility/trino.io/_posts/2022-11-21-trino-summit-2022-recap.md
             ERROR: YOUR SITE COULD NOT BE BUILT:
                    ------------------------------------
                    Could not find post "2022-12-12-trino-summit-2022-lyft-recap" in tag 'post_url'. Make sure the post exists and the name is correct.

```

so restart jekyll/netlify hard and you will see the same problem

**Comment by bitsondatadev** [comment]
I guess we now fail if we use a future date...strange. I'll update the date tomorrow

**Comment by bitsondatadev** [comment]
Date reverted back to 12-12

**Pull request merged**